src/app/pages/my-review/my-review.component.ts

```diff
@@ -0,0 +1,76 @@
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { MatPaginator } from '@angular/material/paginator';
+import {MatSnackBar} from '@angular/material/snack-bar';
+import {MatTableDataSource} from '@angular/material/table';
+import {MatDialog} from '@angular/material/dialog';
+import {UpdateMyReviewComponent} from '../../pages/update-my-review/update-my-review.component';
+
+
+export interface PeriodicElement {
```

Comment:
PeriodicElement? If you're ever going to copy sample code from somewhere, at least change the name of the class. :/

Reply:
Lol, I push to remote too early, fixed.